tools/tx_pull.py

```diff
@@ -107,13 +108,62 @@ def get_parser():
     parser.add_argument(
         '-e', '--email', dest='email',
         help=('Provides an email address used to commit on github if the one '
-              'associated to the github account is not pulbic'))
+              'associated to the GitHub account is not public'))
     parser.add_argument(
         '-t', '--target-org', dest='target',
-        help=('By default, translation are committed in github on OCA. This '
+        help=('By default, translation are committed in GitHub on OCA. This '
               'arg lets you provide an alternative org'))
     return parser
 
+
+def wrap_tx_call(func, args=None, kwargs=None):
```

Comment:
OK, I see. Yeah, it's also possible that way, but if you don't mind, I have invested a lot of time here, so I prefer to keep it this way as this is only a cosmetic change